test/integration/tools/system/pip_manager_test.py

```diff
@@ -0,0 +1,40 @@
+from conan.tools.system import PipEnv
+from unittest.mock import patch
+import mock
+import pytest
+from unittest.mock import PropertyMock
+from conan.errors import ConanException
+from conan.internal.model.settings import Settings
+from conan.test.utils.mocks import ConanFileMock
+
+
+@patch('shutil.which')
+def test_pipenv_conf(mock_shutil_which):
+    # https://github.com/conan-io/conan/issues/11661
+    conanfile = ConanFileMock()
+    conanfile.settings = Settings()
+    mock_shutil_which.side_effect = Exception()
```

Comment:
I think it's cleaner to 


```suggestion
    mock_shutil_which.return_value = "/should/not/be/used"
```

and in the end checking:

```
mock_shutil_which.assert_not_called()
```